src/main/java/eiteam/esteemedinnovation/commons/util/ReflectionHelper.java

```diff
@@ -50,7 +50,7 @@ public static Method getMethod(String methodName, String obfName, Class clazz, C
         try {
             method = clazz.getDeclaredMethod(obfName, params);
         } catch (NoSuchMethodException e) {
-            FMLLog.warning("[EI] Unable to find method " + methodName + " with its obfuscated name. Trying to" +
+            EsteemedInnovation.logger.warn("[EI] Unable to find method " + methodName + " with its obfuscated name. Trying to" +
```

Comment:
`[EI]` should be removed from this, because it says `[esteemedinnovation]` in each log entry